GitUI/Hotkey/HotkeySettingsManager.cs

```diff
@@ -315,6 +315,8 @@ public static HotkeySettings[] CreateDefaultSettings()
                     Hk(RevisionGridControl.Command.CompareToCurrentBranch, Keys.None),
                     Hk(RevisionGridControl.Command.CompareToWorkingDirectory, Keys.Control | Keys.D),
                     Hk(RevisionGridControl.Command.CreateFixupCommit, Keys.Control | Keys.X),
+                    Hk(RevisionGridControl.Command.CreateSquashCommit, Keys.None),
+                    Hk(RevisionGridControl.Command.CreateAmendCommit, Keys.None),
```

Comment:
I've implemented your suggestions but now I noticed that Ctrl+Shift+S is already a hotkey for _Show all branches_.

Reply:
Yes, 'Ctrl+Shift+A' is not free. I must have had focused a wrong control when I checked it.
Add modifier `Alt` yet?

Reply:
Is it worth having such complicated hotkey as a default one? Shouldn't we just leave it empty? I'd say fixup commits (which have a default hotkey) would be used far more than squash or amend ones.

Reply:
> I'd say fixup commits (which have a default hotkey) would be used far more than squash or amend ones.

Valid point.
A "complicated" default hotkey is better than needing to setup one for every single installation IMO. One does not need to remember the hotkey because it is displayed in the menu.
Anyway, which hotkey will you assign?

Reply:
I rather leave the few combinations that is available for something else.

I would use this as I use squash now: Normal commit, toggle amend to get the previous commit message, add amend/squash and modify the text.
(If the commit to change is not the latest, I will copy it first)

Normally, I expect to review and adjust the message when squashing, so fixup/squash/amend is mostly a help for reviewers.

Reply:
OK, so can we agree on leaving the new commands for squash and amend without a default hotkey? I don't mind either way but apparently it would be difficult to figure out some easy to remember hotkey which wouldn't clash with an existing one.